evaporate.js

```diff
@@ -292,7 +307,7 @@
                 priority: 0,
                 onStatusChange: onFileUploadStatusChange,
                 loadedBytes: 0,
-                sizeBytes: (file.file != undefined) ? file.file.size : file.size,
+                sizeBytes: (typeof file.file !== undefined) ? file.file.size : file.size,
```

Comment:
Using typeof now expects a string comparison, so should be `"undefined"`, but there are a lot more backwards compatibility issues than this one. See my PR comment.